### PR TITLE
docs(all): add abbreviation list

### PR DESCRIPTION
**Pull request by takam5f2**
Signed-off-by: Takayuki AKAMINE <takayuki.akamine@tier4.jp>

### CODE REVIEW

File: includes/glossary.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+*[Autoware]: OSS-based self-driving software
+*[CARET]: Chain-Aware ROS Evaluation Tool
+*[DDS]: Data Distribution System
+*[LTTng]: Linux Tracing Tool next generation
+*[OS]: Operating System
+*[ROS]: Robot Operating System \*[TILDE]: Tilde Is Latency Embedding (Integrated Latency Deadline Estimator)
```
Comment (hsgwa) [inline includes/glossary.md:6]: It appears that the line breaks are missing.
```suggestion
*[ROS]: Robot Operating System
*[TILDE]: Tilde Is Latency Embedding (Integrated Latency Deadline Estimator)
```

**Review by hsgwa — CHANGES_REQUESTED** [review]
Are the following lines required for all lines?

```

--8<-- "includes/glossary.md"
```

File: includes/glossary.md (hunk 1 above)
Comment (takam5f2) [reply includes/glossary.md:6]: @hsgwa  
> BTW, are the following lines required for all markdown files?
I found that the following link is helpful us to add abbreviations to all pages. Please wait for a while.

https://github.com/autowarefoundation/autoware-documentation/blob/cc1444548308785ec59d148521377bf45117a246/mkdocs.yaml#L79-L81

**Comment by takam5f2** [comment]
@hsgwa  
Could you check this PR again?

**Review by hsgwa — APPROVED** [review]
LGTM!

**Pull request merged**